### PR TITLE
Feature/76 finish gradle plugin

### PR DESCRIPTION
**Pull request by lheimbs**
This simply adds the new gradle plugin and adds the classpath to the targetClasses in pitest when started from the context menu
Feedback will be added in a later PR

### CODE REVIEW

File: pitmutationmate/gradlew
Comment (brianneoberson) [inline pitmutationmate/gradlew:1]: Why do we have two licenses here?

**Review by brianneoberson — COMMENTED** [review]
Only confusion is concerning the 2 two licenses on the /gradlew files, other than that it looks good to me and works as expected.

File: pitmutationmate/gradlew
Comment (lheimbs) [reply pitmutationmate/gradlew:1]: i meant to take out the MIT one but forgot. as i understood the apache 2.0 license of the gradlew files we need to keep it apache 2.0. I'll remove the MIT parts

**Review by brianneoberson — APPROVED** [review]
Looks good to me. 👍

File: .github/linters/.groovylintrc.json
Comment (brianneoberson) [inline .github/linters/.groovylintrc.json:1]: License is missing

File: .github/linters/.yaml-lint.yml
Comment (brianneoberson) [inline .github/linters/.yaml-lint.yml:1]: License is missing

**Review by brianneoberson — COMMENTED** [review]
Sorry for approving before, but I just noticed the licences are missing for these files.

File: .github/linters/.groovylintrc.json
Comment (lheimbs) [reply .github/linters/.groovylintrc.json:1]: added .github/linters/.groovylintrc.json.license

File: .github/linters/.yaml-lint.yml
Comment (lheimbs) [reply .github/linters/.yaml-lint.yml:1]: added license

**Pull request merged**